themes/bootstrap5/scss/components/search.scss

```diff
@@ -45,18 +45,29 @@ $thumbnail-width-large:  160px !default;
 // Hide these on mobile
 @media (max-width: 767px) {
   #datevispublishDatexWrapper,
-  // Bulk action checkboxes:
+    // Bulk action checkboxes:
```

Comment:
For some reason, your style modifications have indented this line too far.

```suggestion
  // Bulk action checkboxes:
```

Reply:
you are absolutely right